generator/generate-one.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+
+# start  by experimenting local run.
+# note about space consumption: out-of-space testing on cloud shell instance.
+
+# poc with one specified repo - vision
+#cmd line:: ./generate-one.sh -c vision -v 3.1.2 -i google-cloud-vision -g com.google.cloud
+
+while getopts c:v:i:g: flag
+do
+    case "${flag}" in
+        c) client_lib_name=${OPTARG};;
+        v) version=${OPTARG};;
+        i) client_lib_artifactid=${OPTARG};;
+        g) client_lib_groupid=${OPTARG};;
+    esac
+done
+echo "Client Library Name: $client_lib_name";
+echo "Client Library Version: $version";
+echo "Client Library GroupId: $client_lib_groupid";
+echo "Client Library ArtifactId: $client_lib_artifactid";
+
+# setup git
+
+## install bazel - commented out as running on local already with bazel
+#sudo apt-get update
+#sudo apt-get install bazel
+
+WORKING_DIR=`pwd`
+
+
+# get googleapis repo
+git clone https://github.com/googleapis/googleapis.git
+
+# Prepare `gapic-generator-java` with Spring generation ability.
+# If keeping a copy in this repo, this is not needed.
+# Checkout `gapic-generator-java`
+git clone https://github.com/googleapis/gapic-generator-java.git
+# get into gapic and checkout branch to use
+cd gapic-generator-java
+git checkout autoconfig-gen-draft2
+# go back to previous folder
+cd -
+
+
+cd googleapis
+# todo: change to local repo --> gapic
+# very not stable change - todo: change to search and replace.
```

Comment:
To future-proof for the time being, maybe we can check out googleapis on a specific branch or commit in case this file gets updated?